test/functional/http/post/4.multisig.js

```diff
@@ -0,0 +1,530 @@
+					badTransactions.push(transaction);
+				});
+			});
+		});
+
+		describe('lifetime', function () {
+
+			it('using greater than maximum(' + constants.multisigConstraints.lifetime.maximum + ') should fail', function () {
+				transaction = node.lisk.multisignature.createMultisignature(scenarios.regular.account.password, null, scenarios.regular.keysgroup, constants.multisigConstraints.lifetime.maximum + 1, 2);
+
+				return sendTransactionPromise(transaction).then(function (res) {
+					node.expect(res).to.have.property('success').to.be.not.ok;
+					node.expect(res).to.have.property('message').to.equal('Invalid transaction body - Failed to validate multisignature schema: Value ' + (constants.multisigConstraints.lifetime.maximum + 1) + ' is greater than maximum ' + constants.multisigConstraints.lifetime.maximum);
+					badTransactions.push(transaction);
+				});
+			});
+
+			it('using less than minimum(' + constants.multisigConstraints.lifetime.minimum + ') should fail', function () {
+				transaction = node.lisk.multisignature.createMultisignature(scenarios.regular.account.password, null, scenarios.regular.keysgroup, constants.multisigConstraints.lifetime.minimum - 1, 2);
+
+				return sendTransactionPromise(transaction).then(function (res) {
+					node.expect(res).to.have.property('success').to.be.not.ok;
+					node.expect(res).to.have.property('message').to.equal('Invalid transaction body - Failed to validate multisignature schema: Value ' + (constants.multisigConstraints.lifetime.minimum - 1) + ' is less than minimum ' + constants.multisigConstraints.lifetime.minimum);
+					badTransactions.push(transaction);
+				});
+			});
+		});
+	});
+
+	describe('transactions processing', function () {
+
+		it('with no_funds scenario should fail', function () {
+			transaction = node.lisk.multisignature.createMultisignature(scenarios.no_funds.account.password, null, scenarios.no_funds.keysgroup, 1, 2);
+
+			return sendTransactionPromise(transaction).then(function (res) {
+				node.expect(res).to.have.property('success').to.not.be.ok;
+				node.expect(res).to.have.property('message').to.equal('Account does not have enough LSK: ' + scenarios.no_funds.account.address + ' balance: 0');
+				badTransactions.push(transaction);
+			});
+		});
+
+		it('with minimal_funds scenario should be ok', function () {
+			transaction = node.lisk.multisignature.createMultisignature(scenarios.minimal_funds.account.password, null, scenarios.minimal_funds.keysgroup, 1, 2);
+
+			return sendTransactionPromise(transaction).then(function (res) {
+				node.expect(res).to.have.property('success').to.be.ok;
+				node.expect(res).to.have.property('transactionId').to.equal(transaction.id);
+				scenarios.minimal_funds.transaction = transaction;
+			});
+		});
+
+		it('using valid params regular scenario (3,2) should be ok', function () {
```

Comment:
Provide description at the beginning of the file - 3 members, 2 min signatures required

Reply:
Small comment at the top for each Scenario. It can be reviewed when we do the refactor.